index.js

```diff
@@ -46,12 +50,12 @@ function attachToBody (options, req, reply, next) {
   })
 }
 
-function defaultConsumer (field, file, filename, encoding, mimetype, body) {
+function defaultConsumer (field, file, filename, encoding, mimetype, body, fileIndex) {
   const fileData = []
   file.on('data', data => { fileData.push(data) })
-  file.on('limit', () => { body[field].limit = true })
+  file.on('limit', () => { body[field][fileIndex].limit = true })
```

Comment:
There should be no need for `fileIndex`. It's always the last file in the list.

Reply:
Do you mean that we should define it inside function instead of passing as an argument? Because in this particular line I'm not sure that it will be the last file in array. As well as in next line.

Reply:
It it always the last file. The full stream must be consumed for the next one to start.

Reply:
Got it. Thanks.